BuildingOgre.md

```diff
@@ -66,7 +77,8 @@ these will enable input handling in the SampleBrowser, the X11 ConfigDialog and
 * Doxygen: http://doxygen.org
 * Cg: http://developer.nvidia.com/object/cg_toolkit.html
 * Remotery: https://github.com/Celtoys/Remotery
-* %Assimp: https://www.assimp.org/
```

Comment:
please keep the "%" - it is needed as the page is also used by Doxygen:
https://ogrecave.github.io/ogre/api/latest/building-ogre.html#autotoc_md259

Reply:
Thanks for the feedback. Changed.